vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SlideManager.java

```diff
@@ -46,4 +54,27 @@ public ISlide getSlide(String slideId) {
     public void updateSlide(Slide slide) {
         slideRepo.save((Slide) slide);
     }
+    
+    @Override
+    public void deleteSlideById(String slideId) throws SlideDoesNotExistException {
+        try {
+            slideRepo.delete((Slide) getSlide(slideId));
+        } catch (IllegalArgumentException | EmptyResultDataAccessException exception) {
```

Comment:
I don't think the delete method throws an emptyresultdataaccessexception. Are you sure about that? the javadoc says otherwise I believe. Why do you catch that here?